nss-day-cohort-29/welcome-to-nashville-cyclones
# Added save button functionality

# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.


## Type of change



# Testing Instructions for Change Made

1. `git fetch --all`
1. `git checkout brj-project-start`
1. run grunt
1. check that save button prints to itinerary for all search fields


# Checklist:
